extensions/notification/queueProcessor/QueueProcessor.js

```diff
@@ -142,22 +145,31 @@ class QueueProcessor extends EventEmitter {
                     queueProcessor: this.processKafkaEntry.bind(this),
                     logConsumerMetricsIntervalS,
                 });
-                this._consumer.on('error', () => { });
+                this._consumer.on('error', err => {
+                    this.logger.error('error starting notification consumer',
+                    { method: 'QueueProcessor.start', error: err.message });
+                    // crash if got error at startup
+                    if (!this.isReady()) {
```

Comment:
Should we use `this._consumer.isReady()` instead of adding a new method that does the same ?

Reply:
I copied exactly the required changes coming from ZENKO branches so there would be no conflict with waterflow

Reply:
It seems simpler like that